CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## 0.0.1 -
+
+### Added
+  - Initial index.js.
```

Comment:
Remove indents.
Do these lines cover the whole commit or would general shorter comment work better here?

Reply:
Removed: https://github.com/digitalbazaar/bedrock-payment-paypal/pull/2/commits/42a81daf80556f5ef82c3744e34cb7533e466d79